docs/madr/decisions/072-computed-api-endpoints.md

```diff
@@ -0,0 +1,49 @@
+# Computed API Endpoints
+
+* Status: accepted
+
+## Context and Problem Statement
+
+There are two types of API endpoints in Kuma:
+
+- Resource endpoints - These are endpoints that leads to direct reads to the database. They map to Kuma resources. For example: `GET /mesh/{mesh}/dataplanes/{dataplane}`.
+- Computed endpoints - These are endpoints that provide computed data usually using existing resources. For example: `GET /mesh/{mesh}/dataplanes/{dataplane}/_overview` 
+  or `GET /mesh/{mesh}/dataplanes/{dataplane}/_metrics`.
+
+Computed endpoints represent runtime and composed state, this state is not stored in the database and there's no way to update it from the API.
+
+This MADR provides a convention for computed endpoints. The goal is to provide a consistent way to define and document computed endpoints.
```

Comment:
might be worth mentioning "computed" parts of the resources, like `status` or `warnings`

Reply:
Yes done